saleor/payment/gateways/stripe/forms.py

```diff
@@ -0,0 +1,74 @@
+from django import forms
+from django.forms.utils import flatatt
+from django.forms.widgets import HiddenInput
+from django.utils.html import format_html
+from django.utils.translation import pgettext_lazy
+
+from ...forms import PaymentForm
+from .utils import get_amount_for_stripe
+
+CHECKOUT_SCRIPT_URL = 'https://checkout.stripe.com/checkout.js'
+CHECKOUT_DESCRIPTION = pgettext_lazy(
+    'Stripe payment gateway description', 'Total payment')
+
+
+class StripeCheckoutWidget(HiddenInput):
+
+    def __init__(self, payment, gateway_params, *args, **kwargs):
+        attrs = kwargs.get('attrs', {})
+        kwargs['attrs'] = {
+            'class': 'stripe-button',
+            'src': CHECKOUT_SCRIPT_URL,
+            'data-key': gateway_params.get('public_key'),
+            'data-amount': get_amount_for_stripe(
+                payment.total, payment.currency),
+            'data-name': gateway_params.get('store_name'),
+            'data-image': gateway_params.get('store_image'),
+            'data-description': CHECKOUT_DESCRIPTION,
+            'data-currency': payment.currency,
+            'data-locale': gateway_params.get('local'),
+            'data-allow-remember-me': gateway_params.get('remember_me'),
+            'data-billing-address': 'true' if gateway_params.get(
+                'enable_billing_address') else 'false',
+            'data-zip-code': 'true' if gateway_params.get(
+                'enable_billing_address') else 'false',
+            'data-shipping-address': 'true' if gateway_params.get(
+                'enable_shipping_address') else 'false',
```

Comment:
Note: there is a hanging comma here (against Saleor's code style).

Reply:
Thanks, all the hanging comma in this PR are removed